packages/panels/src/Panel/Concerns/HasRoutes.php

```diff
@@ -179,7 +179,7 @@ public function getUrl(?Model $tenant = null): ?string
             return $this->getRedirectUrl($tenant);
         }
 
-        if (Route::has($homeRouteName = $this->generateRouteName('home'))) {
+        if ((! $hasTenancy || $tenant) && Route::has($homeRouteName = $this->generateRouteName('home'))) {
```

Comment:
Just for extra clarity where the `!` applies

```suggestion
        if (((! $hasTenancy) || $tenant) && Route::has($homeRouteName = $this->generateRouteName('home'))) {
```